src/site/topic20.rst

```diff
@@ -91,6 +91,16 @@ Recursive
 * If I wanted to call this method, I would start with ``currentIndex`` as ``0``
     * ``recursiveLinearSearch(someNeedle, someHaystack, 0)``
 
+* It is also common to have a *helper* method that makes the first recursive call with the proper values
+
+.. code-block:: java
+    :linenos:
+
+    public static <T> int recursiveLinearSearch(T needle, T[] haystack) {
+        return recursiveLinearSearch(someNeedle, someHaystack, 0);
```

Comment:
This also introduces method overloading -- I don't recall, has that been touched on? Does it need to be?

Reply:
Pretty sure... I unno. Either way, win.